htdocs/accountancy/admin/fiscalyear_card.php

```diff
@@ -34,7 +34,7 @@
 if ($user->socid > 0) {
 	accessforbidden();
 }
-if (empty($user->rights->accounting->fiscalyear->write)) {
+if (empty($user->hasRight('accounting', 'fiscalyear', 'write'))) {
```

Comment:
Do not use empty on $user->hasRight...  
Use  
`if (!$user->hasRight('accounting', 'fiscalyear', 'write')) {`